members/C001056.yaml

```diff
@@ -39,9 +39,8 @@ contact_form:
         required: true
     - javascript:
       - name: char replace
-        values: ["$MESSAGE"]
         selectors: ["#edit-submitted-write-a-brief-message"]
-        commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/(%2[0C])/g, '');}"]
```

Comment:
are we good to lose this old replace logic? should we have keep it around in addition to the new stuff?

Reply:
deleted that to make the js step a bit simpler. Don't feel strongly about keeping it, but I don't think that old logic is needed anymore